main.py

```diff
@@ -0,0 +1,40 @@
+from Modules.Runtime.environment import SAT
+
+if SAT:
+    from Modules.Magnetometer import magnetometer_factory
+    from Modules.Magnetorquer.hBridge import hBridge
+
+from Modules.DataHub.remote import DataHub
+
+import time
+import math
+
+if SAT:
+    sensor = magnetometer_factory.initialize()
+    actuator = hBridge(23, 24, 14, 15)
```

Comment:
Why aren't you using mangetorquer ?

Reply:
He, I wanted to but got stuck while trying to think of a way to implement those two together with the lab system. For now I think I will just remove it (Or rethink about it). Thanks for pointing that out :sunglasses: